### PR TITLE
kubespray_deploy

### PR DESCRIPTION
**Pull request by KostiaLevchenko**
- added method to run ansible scripts
- implemented functionality to deploy kubespray on proxmox virtual machines using post method

### CODE REVIEW

File: kubehub/urls.py [hunk 1]
```diff
@@ -0,0 +1,7 @@
+from django.conf.urls import url
+from . views import kubespray_deploy
+
+
+urlpatterns = [
+    url(r'^run/ansible$', kubespray_deploy.kubespray_deploy, name='run_ansible_script'),
```
Comment (kopachevsky) [inline kubehub/urls.py]: don't need to expose /ansible from public, finally we should have api lile /cluster/create /cluster/destroy etc

Comment (KostiaLevchenko) [reply kubehub/urls.py]: Fixed 61039ed

File: kubespray_deploy/kubespray_deploy_script.sh [hunk 2]
```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
```
Comment (kopachevsky) [inline kubespray_deploy/kubespray_deploy_script.sh]: align naming with api, if we have cluster api, lets have scripts/cluster

**Review by kopachevsky — APPROVED** [review]

File: kubespray_deploy/kubespray_deploy_script.sh (hunk 2 above)
Comment (KostiaLevchenko) [reply kubespray_deploy/kubespray_deploy_script.sh]: Fixed 249493c

**Pull request merged**